tests/components/google/conftest.py

```diff
@@ -92,14 +92,14 @@
 CLIENT_SECRET = "client-secret"
 
 
-@pytest.fixture(name="calendar_access_role")
-def test_calendar_access_role() -> str:
-    """Default access role to use for test_api_calendar in tests."""
+@pytest.fixture
+def calendar_access_role() -> str:
+    """Set default access role to use for test_api_calendar in tests."""
     return "owner"
 
 
-@pytest.fixture
-def test_api_calendar(calendar_access_role: str) -> None:
+@pytest.fixture(name="test_api_calendar")
+def api_calendar(calendar_access_role: str) -> dict[str, Any]:
```

Comment:
Should the return type be a type alias? That way if it ever changes, only the alias would need to be updated instead of all functions which use the fixture.

Reply:
I think it's unlikely to change, and it's a simple type.
If you change the return type, I would assume all the tests need adjusting anyway.